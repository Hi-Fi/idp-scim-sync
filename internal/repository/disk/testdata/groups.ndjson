{"id":"1","name":"group 1","email":"group.1@mail.com","hashCode":"123456789"}
{"id":"2","name":"group 2","email":"group.2@mail.com","hashCode":"987654321"}