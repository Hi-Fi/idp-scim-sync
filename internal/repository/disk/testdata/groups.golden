{"items":2,"hashCode":"123456789","resources":[{"id":"1","name":"group 1","email":"g.1@mail.com","hashCode":"123456789"},{"id":"2","name":"group 2","email":"g.2@mail.com","hashCode":"987654321"}]}